taxi/admin.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Driver, Car, Manufacturer
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (
+        ("Additional information", {"fields": ("license_number",)}),
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ("Additional information", {"fields": ("license_number",)}),
+    )
+
+
+admin.site.register(Manufacturer)
+admin.site.register(Car)
```

Comment:
Use either decorator or function-based approach for registering your models in the `admin.py`